### PR TITLE
fix #213

### LINKED ISSUE #213: Not able to spin up a container for hosting a local website
While trying to spin up the container using the 
`sudo docker run -d --name kubevirtio -p 4000:4000 -v $(pwd):/srv/jekyll jekyll/jekyll jekyll serve --watch` command on RHEL 7 

**OS Version**
```
[root@localhost kubevirt.github.io]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.5 (Maipo)
```
Im getting the error: 
```
.
.
.
Fetching jekyll-paginate 1.1.0
Installing jekyll-paginate 1.1.0
Fetching jekyll-redirect-from 0.15.0
Installing jekyll-redirect-from 0.15.0
There was an error while trying to write to `/srv/jekyll/Gemfile.lock`. It is
likely that you need to grant write permissions for that path.
```
**Docker output**
```
[root@localhost kubevirt.github.io]# docker ps -a
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                      PORTS               NAMES
596e322e7886        jekyll/jekyll       "/usr/jekyll/bin/ent…"   6 minutes ago       Exited (23) 4 minutes ago                       kubevirtio
```

### PR DESCRIPTION
**Pull request by tripledes**
**What this PR does / why we need it**:

 Add :Z for the docker mount on SELinux enabled systems, with it, docker relabels the directory properly and avoids some of the *permission denied* issues.

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Fixes #213 

**Special notes for your reviewer**:

Hold it until @DirectedSoul1 verifies the solution.

### CODE REVIEW

**Comment by iranzo** [comment]
As related issue has been closed, closing this PR

**Comment by codificat** [comment]
> Automatic relabeling can be problematic if used against some common system folder as it can break further access to other apps.
> 
> I would put a note so that user takes that into account and can decide

Good suggestion to add some comment explaining the command and its implications.

However:

> NACK from me, as this is used by users willing to test documentation changes/development other users will just visit the website

Note that the proposed change is repalcing a `chcon -R` command... in that sense, I believe it's not making things worse.

But it is a good idea to explian what you are doing and why, and highlight precautions.

**Comment by tripledes** [comment]
> > Automatic relabeling can be problematic if used against some common system folder as it can break further access to other apps.
> > I would put a note so that user takes that into account and can decide
> 
> Good suggestion to add some comment explaining the command and its implications.
> 
> However:
> 
> > NACK from me, as this is used by users willing to test documentation changes/development other users will just visit the website
> 
> Note that the proposed change is repalcing a `chcon -R` command... in that sense, I believe it's not making things worse.

It's actually doing it properly:

    This will label the content inside the container with the exact MCS label that the container will run with, basically it runs chcon -Rt svirt_sandbox_file_t -l s0:c1,c2 <directory> where s0:c1,c2 differs for each container.

> 
> But it is a good idea to explian what you are doing and why, and highlight precautions.

**Comment by tripledes** [comment]
Note added.

File: README.md [hunk 1]
```diff
@@ -110,17 +110,23 @@ Under the Community page, depending on your content settings, you will be able t
...
-```
+* On a SELinux enabled OS:
 
-### Run a jekyll container
+    ```console
+    cd kubevirt.github.io
+    sudo docker run -d --name kubevirtio -p 4000:4000 -v $(pwd):/srv/jekyll:Z jekyll/jekyll jekyll serve --watch
+    ```
+    
+    **NOTE**: The Z at the end of the volume (-v) allows relabelling of the web contents so it can be written from within the container, like running `chcon -Rt svirt_sandbox_file_t -l s0:c1,c2` yourself.
```
Comment (iranzo) [inline README.md]: I would change web contents for 'the current working directory at the time of execution, ensure to be inside the web content folder before executing it' (or similar idea)

**Comment by rmohr** [comment]
@codificat FYI, it seems like other PRs were merged since prow ran CI on this one. Therefore it is now re-reunning the tests before finally merging.

**Comment by codificat** [comment]
Ah ok :)  thanks. Still nice to play with the bot...
/pony

**Comment by rmohr** [comment]
man, now I have to enable ponies, cats and dogs plugin. :dagger:

**Comment by rmohr** [comment]
Ok, @jparrill configured prow to use `squash` merges. That was disabled on this repo. I enabled it and now prow merged everything. From now on you can use `/lgtm` and `/approve` to merge, *instead* of hitting the `Merge` button. This will ensure that prow definitely always tested everything on latest master.

**Comment by jparrill** [comment]
@rmohr Hey, just curious, was necessary to enable it in other place than here? https://github.com/kubevirt/project-infra/pull/113/files#diff-3818e252b2f35bd93cd7815848eeb1f7R40

**Comment by rmohr** [comment]
`squash` needed to be allowed in the repo settings here.

**Comment by jparrill** [comment]
Ohh you're right :), thanks @rmohr++

**Comment by rmohr** [comment]
@codificat there you go.

**Pull request merged**